orienteer-core/src/main/resources/log4j.properties

```diff
@@ -2,7 +2,7 @@ log4j.appender.Stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.Stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.Stdout.layout.conversionPattern=[%.6p] (%d) %.26c{1}: %m%n
 
-log4j.rootLogger=INFO,Stdout
+log4j.rootLogger=DEBUG,Stdout
```

Comment:
You can change that locally, but default one it's better to keep as INFO.

Reply:
Yes. This is my local changes. I added this file to commit accidentally